comparison.md

```diff
@@ -334,14 +334,14 @@ Help improve our documentation! If you find anything wrong or which could be imp
 				<td><strong>Drush support</strong></td>
 				<td>✅ Full support of all commands</td>
 				<td>⚠️ some commands not supported (like sql-sqlc, php-eval)</td>
-				<td>✅</td>
+				<td>✅ Full support of all commands, aliases automatically provided and updated</td>
```

Comment:
headsup: removed the part `aliases automatically provided and updated` as this is covered in the next row.